webcrypto/subtle_crypto_test.go

```diff
@@ -35,10 +46,10 @@ func TestSubtleCryptoGenerateKey(t *testing.T) {
 	t.Run("successes", func(t *testing.T) {
 		t.Parallel()
 
-		ts := newConfiguredRuntime(t)
+		ts := newWebPlatformTestRuntime(t)
 
 		gotErr := ts.EventLoop.Start(func() error {
-			err := executeTestScripts(ts.VU.Runtime(), "./tests/generateKey", "successes.js")
+			err := executeTestScripts(ts.VU.Runtime(), webPlatformTestSuite+"generateKey", "successes.js")
```

Comment:
I pointed out this in the RSA PR, but a lot of those test seem *very* similar and likely can be a table test instead.